### ISSUE TITLE
Brush Icons - Revert missing icons (for now)

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#4259]
![image](https://github.com/Bforartists/Bforartists/assets/35357409/5c9488ab-f2d8-4f5b-9371-68c99a676ab4)

Brush Assets is changing the whole brush game, and they removed the icons in the main premturely. Now we have no icons if at all. So we gotta figure out their new system eventually - and update icons or atleast keep them for now till they fully merge in the Brush Assets with the new icons they have. 

Here is the old code in `rna_brush.cc`
```
const EnumPropertyItem rna_enum_brush_sculpt_tool_items[] = {
    {SCULPT_TOOL_DRAW, "DRAW", ICON_BRUSH_SCULPT_DRAW, "Draw", ""},
    {SCULPT_TOOL_DRAW_SHARP, "DRAW_SHARP", ICON_BRUSH_SCULPT_DRAW, "Draw Sharp", ""},
    {SCULPT_TOOL_CLAY, "CLAY", ICON_BRUSH_CLAY, "Clay", ""},
    {SCULPT_TOOL_CLAY_STRIPS, "CLAY_STRIPS", ICON_BRUSH_CLAY_STRIPS, "Clay Strips", ""},
    {SCULPT_TOOL_CLAY_THUMB, "CLAY_THUMB", ICON_BRUSH_CLAY_STRIPS, "Clay Thumb", ""},
    {SCULPT_TOOL_LAYER, "LAYER", ICON_BRUSH_LAYER, "Layer", ""},
    {SCULPT_TOOL_INFLATE, "INFLATE", ICON_BRUSH_INFLATE, "Inflate", ""},
    {SCULPT_TOOL_BLOB, "BLOB", ICON_BRUSH_BLOB, "Blob", ""},
    {SCULPT_TOOL_CREASE, "CREASE", ICON_BRUSH_CREASE, "Crease", ""},
    RNA_ENUM_ITEM_SEPR,
    {SCULPT_TOOL_SMOOTH, "SMOOTH", ICON_BRUSH_SMOOTH, "Smooth", ""},
    {SCULPT_TOOL_FLATTEN, "FLATTEN", ICON_BRUSH_FLATTEN, "Flatten", ""},
    {SCULPT_TOOL_FILL, "FILL", ICON_BRUSH_FILL, "Fill", ""},
    {SCULPT_TOOL_SCRAPE, "SCRAPE", ICON_BRUSH_SCRAPE, "Scrape", ""},
    {SCULPT_TOOL_MULTIPLANE_SCRAPE,
     "MULTIPLANE_SCRAPE",
     ICON_BRUSH_SCRAPE,
     "Multi-plane Scrape",
     ""},
    {SCULPT_TOOL_PINCH, "PINCH", ICON_BRUSH_PINCH, "Pinch", ""},
    RNA_ENUM_ITEM_SEPR,
    {SCULPT_TOOL_GRAB, "GRAB", ICON_BRUSH_GRAB, "Grab", ""},
    {SCULPT_TOOL_ELASTIC_DEFORM, "ELASTIC_DEFORM", ICON_BRUSH_GRAB, "Elastic Deform", ""},
    {SCULPT_TOOL_SNAKE_HOOK, "SNAKE_HOOK", ICON_BRUSH_SNAKE_HOOK, "Snake Hook", ""},
    {SCULPT_TOOL_THUMB, "THUMB", ICON_BRUSH_THUMB, "Thumb", ""},
    {SCULPT_TOOL_POSE, "POSE", ICON_BRUSH_GRAB, "Pose", ""},
    {SCULPT_TOOL_NUDGE, "NUDGE", ICON_BRUSH_NUDGE, "Nudge", ""},
    {SCULPT_TOOL_ROTATE, "ROTATE", ICON_BRUSH_ROTATE, "Rotate", ""},
    {SCULPT_TOOL_SLIDE_RELAX, "TOPOLOGY", ICON_BRUSH_GRAB, "Slide Relax", ""},
    {SCULPT_TOOL_BOUNDARY, "BOUNDARY", ICON_BRUSH_GRAB, "Boundary", ""},
    RNA_ENUM_ITEM_SEPR,
    {SCULPT_TOOL_CLOTH, "CLOTH", ICON_BRUSH_SCULPT_DRAW, "Cloth", ""},
    {SCULPT_TOOL_SIMPLIFY, "SIMPLIFY", ICON_BRUSH_DATA, "Simplify", ""},
    {SCULPT_TOOL_MASK, "MASK", ICON_BRUSH_MASK, "Mask", ""},
    {SCULPT_TOOL_DRAW_FACE_SETS, "DRAW_FACE_SETS", ICON_BRUSH_MASK, "Draw Face Sets", ""},
    {SCULPT_TOOL_DISPLACEMENT_ERASER,
     "DISPLACEMENT_ERASER",
     ICON_BRUSH_SCULPT_DRAW,
     "Multires Displacement Eraser",
     ""},
    {SCULPT_TOOL_DISPLACEMENT_SMEAR,
     "DISPLACEMENT_SMEAR",
     ICON_BRUSH_SCULPT_DRAW,
     "Multires Displacement Smear",
     ""},
    {SCULPT_TOOL_PAINT, "PAINT", ICON_BRUSH_SCULPT_DRAW, "Paint", ""},
    {SCULPT_TOOL_SMEAR, "SMEAR", ICON_BRUSH_SCULPT_DRAW, "Smear", ""},
    {0, nullptr, 0, nullptr, nullptr},
};

const EnumPropertyItem rna_enum_brush_vertex_tool_items[] = {
    {VPAINT_TOOL_DRAW, "DRAW", ICON_BRUSH_MIX, "Draw", ""},
    {VPAINT_TOOL_BLUR, "BLUR", ICON_BRUSH_BLUR, "Blur", ""},
    {VPAINT_TOOL_AVERAGE, "AVERAGE", ICON_BRUSH_BLUR, "Average", ""},
    {VPAINT_TOOL_SMEAR, "SMEAR", ICON_BRUSH_BLUR, "Smear", ""},
    {0, nullptr, 0, nullptr, nullptr},
};

const EnumPropertyItem rna_enum_brush_weight_tool_items[] = {
    {WPAINT_TOOL_DRAW, "DRAW", ICON_BRUSH_MIX, "Draw", ""},
    {WPAINT_TOOL_BLUR, "BLUR", ICON_BRUSH_BLUR, "Blur", ""},
    {WPAINT_TOOL_AVERAGE, "AVERAGE", ICON_BRUSH_BLUR, "Average", ""},
    {WPAINT_TOOL_SMEAR, "SMEAR", ICON_BRUSH_BLUR, "Smear", ""},
    {0, nullptr, 0, nullptr, nullptr},
};

const EnumPropertyItem rna_enum_brush_image_tool_items[] = {
    {PAINT_TOOL_DRAW, "DRAW", ICON_BRUSH_TEXDRAW, "Draw", ""},
    {PAINT_TOOL_SOFTEN, "SOFTEN", ICON_BRUSH_SOFTEN, "Soften", ""},
    {PAINT_TOOL_SMEAR, "SMEAR", ICON_BRUSH_SMEAR, "Smear", ""},
    {PAINT_TOOL_CLONE, "CLONE", ICON_BRUSH_CLONE, "Clone", ""},
    {PAINT_TOOL_FILL, "FILL", ICON_BRUSH_TEXFILL, "Fill", ""},
    {PAINT_TOOL_MASK, "MASK", ICON_BRUSH_TEXMASK, "Mask", ""},
    {0, nullptr, 0, nullptr, nullptr},
};
```
And here:
```
const EnumPropertyItem rna_enum_brush_curves_sculpt_tool_items[] = {
    {CURVES_SCULPT_TOOL_SELECTION_PAINT,
     "SELECTION_PAINT",
     ICON_BRUSH_PAINT_SELECT,
     "Paint Selection",
     ""},
    RNA_ENUM_ITEM_SEPR,
    {CURVES_SCULPT_TOOL_ADD, "ADD", ICON_BRUSH_CURVES_ADD, "Add", ""},
    {CURVES_SCULPT_TOOL_DELETE, "DELETE", ICON_BRUSH_CURVES_DELETE, "Delete", ""},
    {CURVES_SCULPT_TOOL_DENSITY, "DENSITY", ICON_BRUSH_CURVES_DENSITY, "Density", ""},
    RNA_ENUM_ITEM_SEPR,
    {CURVES_SCULPT_TOOL_COMB, "COMB", ICON_BRUSH_CURVES_COMB, "Comb", ""},
    {CURVES_SCULPT_TOOL_SNAKE_HOOK, "SNAKE_HOOK", ICON_BRUSH_CURVES_SNAKE_HOOK, "Snake Hook", ""},
    {CURVES_SCULPT_TOOL_GROW_SHRINK,
     "GROW_SHRINK",
     ICON_BRUSH_CURVES_GROW_SHRINK,
     "Grow / Shrink",
     ""},
    {CURVES_SCULPT_TOOL_PINCH, "PINCH", ICON_BRUSH_CURVES_PINCH, "Pinch", ""},
    {CURVES_SCULPT_TOOL_PUFF, "PUFF", ICON_BRUSH_CURVES_PUFF, "Puff", ""},
    {CURVES_SCULPT_TOOL_SMOOTH, "SMOOTH", ICON_BRUSH_CURVES_SMOOTH, "Smooth", ""},
    {CURVES_SCULPT_TOOL_SLIDE, "SLIDE", ICON_BRUSH_CURVES_SLIDE, "Slide", ""},
    {0, nullptr, 0, nullptr, nullptr},
};
```

<details>
<summary>Details</summary>

SHA-1: fd5a790f36071d79315aed6e504956f8edaea717

* UI: Remove icon usage in brush tool RNA enums

With brush assets, these tool-type icons will be removed completely.
In main already though, it makes some sense to remove the usage of
the icons in the brush tool type enum items. These icons are quite
different than the others and don't look good with a small size anyway.

As far as I can tell this doesn't make a difference in the UI anyway,
since the enum menu selector exposed in the "Brushes" panel didn't
show the icons anyway.

Pull Request: https://projects.blender.org/blender/blender/pulls/121364

</details>

<details>
<summary>More Details</summary>

SHA-1: 39f0e3c0f4bf35410e6d9cf094ac198e0d1d65cf

* UI: Remove more use of brush tool type icons in enum

Missing from fd5a790f36071d79315aed6e504956f8edaea717

</details>

**Comment by ReinerBforartists** [comment]
Does Blender have icons here?

**Comment by Draise14** [comment]
![image](https://github.com/Bforartists/Bforartists/assets/35357409/400083d1-57df-4d78-a2d9-4066a80554e8)

No.

**Comment by Draise14** [comment]
Looks like they reverted it, so will see if this is fixed in the next merge.

<details>
<summary>Details</summary>

SHA-1: 4e622fbccda39028498f8117227f23e1a2b9a13e

* Revert "UI: Remove icon usage in brush tool RNA enums"

This reverts commit fd5a790f36071d79315aed6e504956f8edaea717.

Fixes #121700


</details>

**Comment by Draise14** [comment]
Fixed with the merge
